pkgs/top-level/aliases.nix

```
@@ -738,6 +738,9 @@ mapAliases ({
 
   gcc-snapshot = throw "Marked as broken for >2 years, additionally this 'snapshot' pointed to a fairly old one from gcc7.";
 
+  # added 2020-10-13
+  todolist = ultralist;
```

Can you please throw an error?

---

Done